src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+:root {
+  --active-color: #00acdc;
+
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif, serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)